packages/frontend/src/components/wallet/Wallet.js

```diff
@@ -351,6 +350,13 @@ export function Wallet({ tab, setTab }) {
                     accountId={accountId}
                 />
             }
+            {createCustomName &&
+                <CreateCustomNameModal
+                    onClose={() => dispatch(setCreateCustomName(false))}
+                    isOpen={createCustomName}
+                    accountId='bob.near'
```

Comment:
Should this be hard coded to `bob.near`?

Reply:
Yes - though might be nice with a fading carousel style that show many difference options cc @corwinharrell

Reply:
Once we implement Torus it'd be nice to pull the social login info and construct a recommended account id based on name, email, etc